src/main/java/nus/climods/logic/commands/PrereqsCommand.java

```diff
@@ -0,0 +1,23 @@
+package nus.climods.logic.commands;
+
+import nus.climods.logic.commands.exceptions.CommandException;
+import nus.climods.model.Model;
+
+public class PrereqsCommand extends Command {
+    public static final String COMMAND_WORD = "deps";
```

Comment:
I'm not sure if `deps` would be intuitive for the user, was thinking we can use `prereq` (or `prereqs`) instead? What do you think?

Reply:
I agree deps isn't as intuitive but I also felt that `prereq` or `prereqs` is a bit long compared to our other commands. If we don't plan on showing preclusions as well how about `pre`?

Reply:
im actually not too sure about this, what do the rest think? @sarrrdin @SeekSaveServe @zupey

Reply:
I personally would prefer `prereq` because it is more intuitive despite that it is slightly verbose. Unless our command word is `pq` hmm

Reply:
google is telling me that preq is the most common way to abbreviate prerequisite. As a good CS student i trust my google searches and so I vote for `preq`

Reply:
i think `preq` is good!

Reply:
'preq' sounds good to me too